### PR TITLE
ames: remove num-live from pump-metrics

### PR DESCRIPTION
**Pull request by yosoyubik**
Tested this on two different live moons and saw no communication issues: they can talk to each other, their sponsor, install apps, join/leave/rejoin groups.

Addresses decrement-underflow error seen in #6075 and https://github.com/urbit/urbit/issues/5979#issuecomment-1275723988

Note: this updates `%lull`

### CODE REVIEW

**Review by belisarius222 — DISMISSED** [review]
LGTM.  I'm sad we have to do this, since it worsens congestion control asymptotics, but it is the fastest way to fix the bug.  A good goal is a refactored Ames that will be so much easier to reason about that tracking num-live won't be bug-prone.

**Comment by zalberico** [comment]
@belisarius222 and @yosoyubik - I want to merge fixes like this once approved into `next/arvo` (and deploy them on the internal canary) unless there's some specific special reason not to.

**Comment by zalberico** [comment]
Does this also resolve #6050 or is that something different?

**Comment by yosoyubik** [comment]
@zalberico #6050 is another issue (we are getting a nack response for a message we don't have in our state anymore). 

Re this PR, once I revert https://github.com/urbit/urbit/pull/6076/commits/b2a0f2fd304f961e4db89499aedefe0be95e6239 (se we don't update %lull yet), it'll be ready to be merged into next/arvo

**Comment by yosoyubik** [comment]
@belisarius222 Just reverted the commit that updated %lull and updated %ames' state—num-live is still in the pump but it's unused. This doesn't reload anything besides the vane itself.

@zalberico Once I get an approval, I'll merge into next/arvo.

**Comment by zalberico** [comment]
Great - thanks, the next part of this is we need to be deploying the tip of next/arvo to our internal canary weekly (and we need kernel devs to be sourcing their OTAs from that canary)

**Comment by yosoyubik** [comment]
@zalberico I can take care of deploying to ~binnec (it's that still the dev canary?)—what has the deployment cadence been to give people a heads up?. Also, do we want to deploy both ames PRs, or do we want a more granular deployment?

**Comment by yosoyubik** [comment]
@belisarius222 checking that this looks good to go into next/arvo—latest commit don't update %lull, so they leave num-live unused (we can take care of the state migration later with some of the other %ames refactoring work, like removing krocs, for example, since it's not really used anyway)

**Review by belisarius222 — APPROVED** [review]
lgtm

**Pull request merged**
